WordPress/twentytwentyfour
# Clients pattern adjustments

**Description**

Fixes https://github.com/WordPress/twentytwentyfour/issues/246 with the adjustments proposed by Beatriz

**Screenshots**



<img width="1426" alt="Screenshot 2023-09-07 at 16 55 11" src="https://github.com/WordPress/twentytwentyfour/assets/252415/1f92a605-7942-4968-82c1-1421d487b6d0">

**Testing Instructions**

1. Activate TT4 theme.
2. Create a page.
3. Insert the client pattern, confirm the changes.
